### DISCUSSION TITLE
Upload limit on deployed app - 413 error

### DISCUSSION THREAD

**Discussion by chrisnoden** [nuxt/nuxt#19216] [Questions]
Hi

I know this might not be the best forum for this question, but I've googled the hell out of this and can't find a solution.

I deploy my Nuxt v3 apps to DigitalOcean's App platform. I find this works well and it's a real PITA to deploy directly to a Linux server (docs on using pm2 are thin for us beginners to Node).

My latest project takes file uploads. It works really well on dev and I don't need stupid upload limits - a few Mb would be fine. But, when deployed either running in the App platform (a heroku nodejs buildpack) or running `yarn start` on a linux server, I get 413 errors with files a lot smaller than the ones I've tested on dev. (Seems to be under 1Mb the limit).

Anybody know how to increase the file upload/payload limit? All the google answers are for express which doesn't apply to Nuxt v3 of course. I can't find that Nitro has any limits....

Many thanks.

**Comment by andysay** [comment]
no info ?

**Reply by seraphimMM** [reply]
It seems not a bug of Nuxt.
check your nginx config, add this line to your server config

client_max_body_size 5M;
